src/Api/RomeoShop.Backend.Api/Controllers/ImageController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoMapper;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using RomeoShop.Backend.Contracts.Image;
+using RomeoShop.Backend.Cqrs.Commands.Image;
+using RomeoShop.Backend.Cqrs.Queries.Image;
+using RomeoShop.Backend.Dtos.Image;
+
+namespace RomeoShop.Backend.Api.Controllers
+{
+    [Route("api/v1/image")]
+    [ApiController]
+    public class ImageController : Controller
```

Comment:
Controller используется для MVC проектов. Для API - ControllerBase

```suggestion
    [Route("api/v1/image")]
    public class ImageController : ControllerBase
```